start-site/src/main/resources/application.yml

```diff
@@ -1020,6 +1020,21 @@ initializr:
             - rel: guide
               href: https://github.com/spring-cloud-samples/bookstore-service-broker
               description: Using Spring Cloud Open Service Broker
+        - name: App Broker
+          id: app-broker
+          compatibilityRange: "[2.2.0.RELEASE,2.4.0-M1)"
+          description: Framework for building Spring Boot applications that implement the Open Service Broker API and deploy applications as brokered services.
+          groupId: org.springframework.cloud
+          artifactId: spring-cloud-starter-app-broker-cloudfoundry
```

Comment:
Looking at the current description and name, it is not obvious at all this is Cloudfoundy specific (as I am guessing given the name of the starter).

Reply:
App Broker offers an abstraction, but it currently only supports Cloud Foundry. I'll add a bit of clarification.